libr/bin/format/bflt/bflt.h

```diff
@@ -0,0 +1,72 @@
+/* radare - LGPL - Copyright 2016 - Oscar Salvador */
+
+#ifndef BFLT_H
+#define BFLT_H
+
+#include <r_types.h>
+#include <r_util.h>
+#include <r_lib.h>
+#include <r_bin.h>
+
+/* Version 2 */
+#define OLD_FLAT_VERSION	0x00000002L
+#define FLAT_RELOC_TYPE_TEXT	0x0
+#define FLAT_RELOC_TYPE_DATA	0x1
+#define FLAT_RELOC_TYPE_BSS	0x2
+
+struct bflt_relocation {
+#if defined(__BIG_ENDIAN_BITFIELD)
+        ut32 type : 2;
```

Comment:
dont use binary attribues like :2 : 30, ... this is not portable, better use & binary masks when accessing fields